boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0_defconfig

```diff
@@ -15,6 +15,8 @@ CONFIG_CORTEX_M_SYSTICK=y
 CONFIG_GPIO=y
 CONFIG_PINMUX=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=96000000
+CONFIG_ARM_MPU=y
```

Comment:
Strictly speaking @MaureenHelm @agansari 
this core is an M33 so it does not need ARM_MPU to have HW_STACK_PROTECTION set.
But it is OK to enable MPU by default - this will allow enabling User Mode in samples/tests.